packages/neuron-wallet/src/services/live-cells.ts

```diff
@@ -0,0 +1,50 @@
+import { getConnection, In } from 'typeorm'
+import { hexToBytes } from '@nervosnetwork/ckb-sdk-utils'
+import LiveCellEntity from 'database/chain/entities/live-cell'
+
+export default class LiveCellsService {
+  // exclude hasData = true and typeScript != null
+  public static async getBalance(lockHashes: Set<string>): Promise<{
+    liveBalance: Map<string, string>
+    sentBalance: Map<string, string>
+    pendingBalance: Map<string, string>
+  }> {
+    const hashData: Buffer[] = []
+    lockHashes.forEach((lockHash: string) => hashData.push(Buffer.from(hexToBytes(lockHash))))
+
+    const cells: { usedBlockNumber: string, lockHash: Buffer, sumOfCapacity: string }[] = await getConnection()
+      .getRepository(LiveCellEntity)
+      .createQueryBuilder('cell')
+      .select('cell.lockHash', 'lockHash')
+      .addSelect('cell.usedBlockNumber', 'usedBlockNumber')
+      .addSelect('CAST(SUM(CAST(cell.capacity AS UNSIGNED BIG INT)) AS VARCHAR)', 'sumOfCapacity')
+      .where({
+        lockHash: In([...hashData]),
+        data: Buffer.from('0x', 'hex'),
```

Comment:
`Buffer.from('0x', 'hex')` in meaningless, `Buffer.from('0x....', 'hex')` always returns an empty buffer.

This could be replaced by `[]`